stable/pachyderm/templates/NOTES.txt

```diff
@@ -0,0 +1,15 @@
+In order to use Pachyderm, you need to install the Pachyderm command line client: pachctl.
+
+On Linux environments:
+
+$ curl -o /tmp/pachctl.deb -L https://github.com/pachyderm/pachyderm/releases/download/v1.6.6/pachctl_1.6.6_amd64.deb && sudo dpkg -i /tmp/pachctl.deb
+
+And on Mac:
+
+$ brew tap pachyderm/tap && brew install pachyderm/tap/pachctl@1.6
+
+To check that installation was successful, you can try running pachctl help, which should return a list of Pachyderm commands. Please note that the client version should correspond to the pachd service version. For more information please consult: http://pachyderm.readthedocs.io/en/latest/index.html. 
+
+If you install the client on your master node, you should already be able to interact with the service. Also, if you have your kubernetes client properly configured to talk with your remote cluster, you can simply install `pachctl` on your local machine and execute: `pachctl port-forward &`.
```

Comment:
Can we print out a useable pachctl command? e.g. `pachctl port-forward -k -n {{.Release.Namespace}} &` would be useful when pachd is not in the default/configured NS.